src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -156,7 +167,10 @@ def displayPatternAtIndices(
         if self._file_name is None:
             return
 
-        point = Point(indices, file_name=self._file_name)
+        point = Point(indices,
+                      file_name=self._file_name,
+                      nxdata_path=f"{self._nxprocess_path}/result",
```

Comment:
Minor: `Point` could be changed to accept the URI to the `NXData` instead of both the `file_name` and the `nxdata_path`.